aten/src/ATen/native/cuda/AmpKernels.cu

```diff
@@ -156,7 +156,10 @@ void _amp_foreach_non_finite_check_and_unscale_cuda_(TensorList scaled_grads,
 
       // multi_tensor_apply guards onto tensor_lists[0][0], no need to guard explicitly.
       multi_tensor_apply<1>(tensor_lists,
-                            UnaryOpFunctor_<scalar_t>(),
+                            UnaryOpFunctor<scalar_t, 
+                                           /* depth */ 1,
```

Comment:
nit: it looks like everywhere `depth` is equal to `multi_tensor_apply` template argument (and moreover, things would break if it isn't), so maybe use constexpr variable there?